wd_core.au3

```diff
@@ -1428,15 +1441,14 @@ EndFunc   ;==>__WD_Get
 ; Example .......: No
 ; ===============================================================================================================================
 Func __WD_Post($sURL, $sData)
+	$_WD_HTTPRESULT = 0 ; Reseting result of last WinHTTP request
 	Local Const $sFuncName = "__WD_Post"
 	Local $iResult, $sResponseText, $iErr

```

Comment:
why `$iResult` and `$iErr` ?

Reply:
Because that's how I originally coded it.